Source/Wit/Private/Wit/Request/WitRequestSubsystem.cpp

```diff
@@ -317,9 +317,10 @@ void UWitRequestSubsystem::OnRequestProgress(FHttpRequestPtr Request, int32 Byte
 
 	SplitResponseIntoChunks(Content, ChunkedResponses);
 
-	const bool bIsMalformedResponse = (ChunkedResponses.Num() == 0);
-	if (bIsMalformedResponse)
+	const bool bIsDataResponse = (ChunkedResponses.Num() == 0);
```

Comment:
It was previously named malformed because it did not contain json data, now that we are also accepting binary data, I renamed it accordingly, the `== 0` is correct as it differentiates json data from binary data